imagelab/ImgProvider.java

```diff
@@ -26,6 +26,7 @@
     int             pixheight;
     /** Image width in pixels. */
     int             pixwidth;
+    static Thread playThread;
```

Comment:
What is the intent of this class field?  (At the very least, this requires a Javadoc comment.)

Reply:
I have added Javadoc to this and will resubmit PR.